serve-web/src/Repository/OrderRepository.php

```diff
@@ -80,6 +80,15 @@ private function applyFilters(QueryBuilder $qb, array $filters)
             $qb->andWhere('UPPER(c.caseNumber) LIKE :cn')
             ->setParameter('cn', strtoupper(trim($filters['q'])));
         }
+
+        if (
+            array_key_exists('startDate', $filters) &&
+            array_key_exists('endDate', $filters)
+        ) {
+            $qb->andWhere('o.servedAt >= :start AND o.servedAt <= :end')
+                ->setParameter('start', $filters['startDate'])
+                ->setParameter('end', $filters['endDate']);
+        }
```

Comment:
I'm by no means a fan of this solution, however its the simplest temporary solution for now without having to refactor lots of the OrderRepository